docs/vim.md

```diff
@@ -96,25 +96,25 @@ let g:ale_javascript_prettier_options = '--single-quote --trailing-comma es5'
 Prettier extension for [coc.nvim](https://github.com/neoclide/coc.nvim) which requires neovim or vim8.1.
 Install coc.nvim with your favorite plugin manager, such as [vim-plug](https://github.com/junegunn/vim-plug):
 
-```
+```vim
 Plug 'neoclide/coc.nvim', {'do': { -> coc#util#install()}}
 ```
 
 And install coc-prettier by command:
 
-```
+```vim
 CocInstall coc-prettier
 ```
 
 Setup `Prettier` command in your `init.vim` or `.vimrc`
 
-```
+```vim
 command! -nargs=0 Prettier :call CocAction('runCommand', 'prettier.formatFile')
 ```
 
 Update your `coc-settings.json` for languages that you want format on save.
 
-```
+```vim
   "coc.preferences.formatOnSaveFiletypes": ["css", "Markdown"],
 ```
```

Comment:
This one  seems should be json , I'm AFK.

Reply:
Fixed https://github.com/prettier/prettier/pull/7937/commits/c4612f018f8ab5d19b652205392d4eb6c18110bf